src/async-wrap.cc

```diff
@@ -448,6 +451,56 @@ static void DisablePromiseHook(const FunctionCallbackInfo<Value>& args) {
   }, static_cast<void*>(env));
 }
 
+static void AsyncTaskScheduled(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (!args[0]->IsString())
+    return env->ThrowTypeError("first argument must be a string");
+  if (!args[1]->IsNumber())
+    return env->ThrowTypeError("second argument must be a number");
+  if (!args[2]->IsBoolean())
+    return env->ThrowTypeError("third argument must be a boolean");
+
+  Local<String> taskName = args[0]->ToString(env->context()).ToLocalChecked();
+  String::Value taskNameValue(taskName);
+  StringView taskNameView(*taskNameValue, taskNameValue.length());
+
+  int64_t taskId = args[1]->IntegerValue(env->context()).FromJust();
+  void* task = reinterpret_cast<void*>(taskId);
+  bool recurring = args[2]->BooleanValue();
```

Comment:
If you use `BooleanValue()`, you’ll want the overload that takes a `context` argument like above (but again, no need for casting; I’m going to stop commenting that now).